### ISSUE TITLE
False R0201: Method could be a function

### ISSUE THREAD

**Issue by WayneYe** [PyCQA/pylint#3548]
### Steps to reproduce
1. Create a Django sample project using Django 2.2 LTS
2. Locate or implement a `form_valid` function in any class-based view
3. Run pylint check and you will see this failure:
`R0201: Method could be a function (no-self-use)`

### Expected behavior
The django `form_valid` method is neither a classmethod nor a staticmethod, pylint should not report R0201.

### pylint --version output
```
pylint --version
pylint 2.4.4
astroid 2.3.3
Python 3.8.2 (default, Apr 19 2020, 20:32:10)
[GCC 9.2.0]
```

**Comment by PCManticore** [comment]
Can you provide a short code example on how to reproduce this?

**Comment by PCManticore** [comment]
Yeah, I'm going to close this. I don't know what Django's `form_valid` is, but from the looks of it, it seems it was implemented but `self` was not used in its body, so feel free to disable the error in place where you defined the function.
